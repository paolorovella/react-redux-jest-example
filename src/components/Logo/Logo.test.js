import React from 'react';
import renderer from 'react-test-renderer';
import logo from 'src/assets/logo.svg';

import Logo from '.';

it('<Logo /> renders without crashing', () => {
  const tree = renderer.create(
    <Logo src={logo} />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
