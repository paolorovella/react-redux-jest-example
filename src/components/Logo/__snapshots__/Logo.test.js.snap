// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> renders without crashing 1`] = `
<img
  className="sc-bdVaJa kzBKTv"
  src="logo.svg"
/>
`;
